ci/task_whitesource.yaml

```diff
@@ -3,8 +3,8 @@ platform: 'linux'
 image_resource:
   type: registry-image
   source:
-    repository: keppel.eu-de-1.cloud.sap/ccloud/whitesource-agent
-    tag: 20.1.3-kubernikus
+    repository: keppel.eu-de-1.cloud.sap/ccloud/openstack-wss-unified-agent
+    tag: latest
```

Comment:
I didn't test this in concourse because I would have first to inline it into pipeline.yaml but from testing with keppel we probably need to install go and the go from the repositories is to old. Also the agent won't fail if no go is available.